examples/go.mod

```diff
@@ -6,11 +6,11 @@ require (
 	github.com/aws/aws-sdk-go v1.54.18
 	github.com/aws/aws-sdk-go-v2/config v1.27.26
 	github.com/aws/aws-sdk-go-v2/service/iam v1.34.3
-	github.com/pulumi/providertest v0.0.11
+	github.com/pulumi/providertest v0.0.13
 	github.com/pulumi/pulumi-aws/provider/v6 v6.0.0-00010101000000-000000000000
-	github.com/pulumi/pulumi-terraform-bridge/pf v0.39.0
+	github.com/pulumi/pulumi-terraform-bridge/pf v0.40.0
 	github.com/pulumi/pulumi-terraform-bridge/testing v0.0.2-0.20230927165309-e3fd9503f2d3
-	github.com/pulumi/pulumi/pkg/v3 v3.124.1-0.20240716132715-1e77ed263d17
```

Comment:
We need to keep this pinned here for the disk size issue

Reply:
Good catch. One sec.